public/js/angular/controllers.js

```diff
@@ -1,6 +1,6 @@
 angular
-  .module('webmakerApp')
-  .controller('navigationController', ['$scope', '$location', '$routeParams', '$rootScope', 'weblit', 'wmNav', 'CONFIG',
+  .module("webmakerApp")
```

Comment:
I didn't mean "force everything to single quote", you've been changing files both to "all single" and "all double" instead of picking one and sticking to that. We definitely don't want to prescribe which to use, but in this refactor PR let's -as person making the changes- at least convert every file with "you're using two kinds of quotes" to the same quote symbol.